src/pages/contact-us/index.js

```diff
@@ -0,0 +1,124 @@
+import React, { useReducer } from 'react'
+import { css } from 'styled-components'
+import { Button } from '../../components/elements'
+import Input from '../../components/Input'
+import Layout from '../../components/layout'
+import Subtitle from '../../components/Subtitle'
+import {
+  FlexDiv,
+  FlexDivWrap,
+  SidePanel,
+  H1Inverted,
+  HeadingSection,
+} from './styles'
+
+const CHANGE = 'CHANGE'
+
+const fields = [
+  { name: 'name', label: 'Full name' },
+  { name: 'organization', label: 'Company or organisation' },
+  { name: 'email', type: 'email', label: 'Email address' },
+  { name: 'phone-number', type: 'tel', label: 'Phone number' },
+]
+
+const reducer = (s, action) => {
+  switch (action.type) {
+    case CHANGE: {
+      const { key, value } = action
+      return { ...s, [key]: value }
+    }
+    default:
+      return s
+  }
+}
+const Form = () => {
+  const [state, dispatch] = useReducer(reducer, {
+    name: '',
+    org: '',
+    email: '',
+    phone: '',
+  })
+
+  return (
+    <form
+      method="post"
+      name="contact-us"
+      action="/contact-us/success"
+      netlify
+      netlify-honeypot="bot-field"
```

Comment:
Can you link the part of the docs that is referencing this?

Reply:
https://docs.netlify.com/forms/setup/